packages/lockfile-file/test/__snapshots__/write.test.ts.snap

```diff
@@ -18,6 +18,8 @@ packages:
     engines: {node: '>=10', npm: \\"\\\\nfoo\\\\n\\"}
     cpu: [x86]
     os: [darwin]
+    libc:
+      - glibc
```

Comment:
@zkochan looks like this pr merged and publish a new package version of js-yaml, it will be fine here.
i open a pr for it: https://github.com/pnpm/js-yaml/pull/1

Reply:
ok, published